### ISSUE TITLE
Dont work for me, get an error

### ISSUE THREAD

**Issue by FritzHugo3** [munucrafts/Cpp_UE_Get_Thumbnails#1]
I made it like in youre video - than comes "Error could not be compile. Try rebuilding from source manually" (tryed with Unreal 5.1 and 5.5)

any Idea?

**Comment by munucrafts** [comment]
You might need to remove the saved, intermediate,  and binaries folder and then generate the solution again. Also, make sure to install rembg for python. It should work. Let me know if it works for you or not.

**Comment by FritzHugo3** [comment]
I dont find rembg on FAB/Unreal.  rembg in google many different stuff comes out. You say nothing in the tutorial video nor here anything over a rembg plugin. So what I must do? 

For Visual Studio 2022 is Phyton, c++ windows, c++ games installed. (This also never sayed here or in the tutorial video - this information should be stand in the description, also the rembg thing).

Thank you.  

(For me I byed this now https://www.fab.com/listings/fcd659f7-293f-4218-87dd-11921ace2065 but it is not without the chessboard pattern. So it's not really without background, which is why it would still be cool to find out how rembg works)

**Comment by munucrafts** [comment]
Sorry Man..! I have not got around to update the readme file in some time. I will update it with all the process and let you know here. It will be clear to you on how to use it after that.

**Comment by FritzHugo3** [comment]
thx a lot, can you say, in a week, month... so not exact but round about?

**Comment by munucrafts** [comment]
Will do it today only. Don't worry.

**Comment by FritzHugo3** [comment]
Oh wow, verry cool thx and have a nice weekend.

**Comment by munucrafts** [comment]
Hey Fritz, I have updated the README file for you and now there is a Python script you can use to install Rembg as well. Make sure to follow the steps in the README thoroughly. Let me know if you have any trouble. Hope it helps.

**Comment by FritzHugo3** [comment]
That was fast, thank you verry much. 

![grafik](https://github.com/user-attachments/assets/62adee07-5da5-41d4-908a-1142d0e070cf)
So, not so easy or LUL. 

I have absolutely no idea about Phyton or the other stuff. I just know it doesn't work for me. Nobody who has nothing to do with it can do it with the help of the instructions. 

The video showed everything so simply, but unfortunately it's not that simple. And I think that others will also feel overwhelmed by it.

**Comment by munucrafts** [comment]
Hey Fritz, no problem. I will help you with this. Send me your discord ID through the mail. We will connect and help you do this. Let me know if that sounds good. I am available for the next few hours. Let me know if you want my help with this through Discord.

**Comment by FritzHugo3** [comment]
https://discord.gg/VhexXrjGis my Channel 




    Am Samstag, 23. November 2024 um 07:54:06 MEZ hat Munucrafts ***@***.***> Folgendes geschrieben:  
 
 


Hey Fritz, no problem. I will help you with this. Send me your discord ID through the mail. We will connect and help you do this. Let me know if that sounds good.

**Comment by munucrafts** [comment]
Its showing this invite link is expired.

**Comment by FritzHugo3** [comment]
Mhh strange, it woukd be work 7 days.
Account discord: FritzHugo3
Old number 1565 - Discord has chance the system
https://discord.gg/tP4uKKEH

or give me your name if this dont work. 




    Am Samstag, 23. November 2024 um 18:08:19 MEZ hat Munucrafts ***@***.***> Folgendes geschrieben:  
 
 


Its showing this invite link is expired. Just send me your ID and I will send you a request there.

**Comment by FritzHugo3** [comment]
Why you close it. it doesnt work, what you have sayed.

**Comment by munucrafts** [comment]
Hey Fritz, I have updated the README completely and tested the plugin in multiple projects, and it's working for me. If there are build issues in your UE project, I suggest you look for them. If you still have not been able to install Python, then I suggest you look for a few tutorials and some research, and also use ChatGPT to resolve it. 

I am closing it because your issue is not related to the plugin but rather the Python, pip, and rembg installation and how you are doing it.

**Comment by FritzHugo3** [comment]
I have pyhton installed totaly perfect for all other things who use Pyhton. I programm now with pyhton and all works only not this code. I dont know. But if you mean it is me ok. I can only say, all other works fine. 
For years I was self-employed in PC service, gave lessons, sold hardware... I am now a software developer myself. What can I say about that. I'd be interested to know how many others have managed to make it work, but let's leave it at that, it's no use. 

I build my own software, so I know it will work. 




    Am Donnerstag, 12. Dezember 2024 um 17:32:47 MEZ hat Munucrafts ***@***.***> Folgendes geschrieben:  
 
 


Hey Fritz, I have updated the README completely and tested the plugin in multiple projects, and it's working for me. If there are build issues in your UE project, I suggest you look for them. If you still have not been able to install Python, then I suggest you look for a few tutorials and some research, and also use ChatGPT to resolve it.

I am closing it because your issue is not related to the plugin but rather the Python, pip, and rembg installation and how you are doing it.

**Comment by munucrafts** [comment]
Hey Fritz, no need to get personal here. I am just saying I have used this plugin in multiple projects including at work and it seems to work fine all the time. Even if rembg is not installed it still works except it does not remove the image background. So, I am not sure what is causing you to be unable to use it. From your screenshots, I concluded that it is a Python, pip, rembg installation issue which is mentioned in the cmd image that you have shared.
